dask_sql/physical/rel/custom/predict.py

```diff
@@ -59,7 +60,13 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
 
         model, training_columns = context.schema[schema_name].models[model_name]
         df = context.sql(sql_select)
-        prediction = model.predict(df[training_columns])
+        part = df[training_columns]
+        output_meta = model.predict_meta
+        if part.shape[0].compute() == 0 and output_meta is not None:
```

Comment:
`compute` is needed on the Delayed object. I believe that right now, `output_meta` will always be `[]`

Reply:
You dont need to compute for this to  do this, we can do it lazily too.